api/types/stores.go

```diff
@@ -18,3 +18,9 @@ type SockStore interface {
 	GetSockByID(sockID int) (*Sock, error)
 	GetSockVariants(sockID int) ([]SockVariant, error)
 }
+
+type OrderStore interface {
+    OrderExistsByID(orderID int) (bool, error)
+    UpdateOrderAddress(orderID int, address UpdateAddressRequest) error
+	LogOrderUpdate(orderID, adminID int, message string) error
```

Comment:
nit: reformat looks like a space is missing

Reply:
Fixed I think? I don't see any weird spacing on my end
<img width="729" alt="image" src="https://github.com/user-attachments/assets/e90ccdd6-a68c-4dd8-9ef1-be9e979d862a">

Reply:
weird. maybe it was because I was on my phone then idk. Looks good tho